mcodingbot/plugins/reactions.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+import crescent
+import hikari
+
+RUST_REGEX = re.compile(r"\brust\b", flags=re.I)
+
+plugin = crescent.Plugin("reactions")
+
+
+@plugin.include
+@crescent.event
+async def on_message(event: hikari.MessageCreateEvent) -> None:
+    message = event.message
+
+    if message.type == hikari.MessageType.GUILD_MEMBER_JOIN:
+        await message.add_reaction("👋")
+        return
+
+    if not message.content:
+        return
+
+    if RUST_REGEX.search(message.content) and "🚀" in message.content:
```

Comment:
Is regex actually faster? Why not "rust" in message.content?

Reply:
i need `\b` from regex to make it feel right

Reply:
Gotcha